README.md

```diff
@@ -1 +1,6 @@
-# Hello_Microverse
\ No newline at end of file
+ Hello_Microverse 
+
```

Comment:
- [ ] Please set up linter checks for this project by following this guide [here](https://github.com/microverseinc/linters-config/tree/master/html-css). Linter is a tool that developers use to ensure that they write standard code that is readable. error-free and properly indented. 

In your commit history, I see that your commit message says Create .github/workflow/linter.yml file, but it is not in your repo, please confirm and fix it. 


![image](https://github.com/Isaac-ej/Hello_Microverse/assets/84156714/6650ecd4-f166-4f93-adc4-22e62b030ee6)